Dockerfile

```diff
@@ -7,6 +7,6 @@ COPY dockerscripts/docker-entrypoint.sh /usr/bin/docker-entrypoint.sh
 
 ENTRYPOINT ["/usr/bin/docker-entrypoint.sh"]
 
-VOLUME ["/data"]
+VOLUME ["/data","/data1","/data2","/data3","/data4"]
```

Comment:
I think 5 drives is an odd number, better go with 4 drives, however it might be possible to go around this using `-v` flag on `docker run`

Reply:
You are totally right, `-v` works as intended, no need to do this change. I am sorry!